scene/3d/visual_instance.cpp

```diff
@@ -108,6 +108,17 @@ void VisualInstance::_notification(int p_what) {
 			if (_is_vi_visible() && is_physics_interpolated()) {
 				VisualServer::get_singleton()->instance_reset_physics_interpolation(instance);
 			}
+#if defined(DEBUG_ENABLED) && defined(TOOLS_ENABLED)
+			else if (GLOBAL_GET("debug/settings/physics_interpolation/enable_warnings")) {
```

Comment:
Isn't this expensive to query for each notification? Might be worth caching.

Reply:
This isn't too bad (as still relatively rare), but yes I guess we are calling this so often now I should look at caching. If you are able to merge this I can do a separate PR to modify all the calling sites to a cached version. :+1: